packages/backend/tests/discord.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { client } from '../src/configuration/discord';
+import { TokenRequestMessage } from '../src/controllers/discord/templates';
+import { publish, sendTo } from '../src/controllers/discord';
+import { ApiError } from '../src/controllers/error/api.class';
+
+jest.mock('../src/configuration/environment.ts');
+jest.mock('../src/configuration/discord.ts');
+
+describe('discord.ts discordSendTo', () => {
+  let spyDiscordClientUsers: jest.SpyInstance;
+
+  beforeEach(() => {
+    spyDiscordClientUsers = jest.spyOn(client.users.cache, 'get');
+  });
+
+  it('should throw error if user is not in cache', async () => {
+    spyDiscordClientUsers.mockImplementation(() => null);
+    const compareError = new ApiError(409, `User not in discord cache. Send the bot a small 'test' message (via DM) and try again.`);
+    try {
+      await sendTo('1234657890', new TokenRequestMessage(), { key: 123123 });
+    } catch (error) {
+      expect(error).toEqual(compareError);
+    }
+  });
+
+  it('should send message if everything is fine', async () => {
+    const mockDiscordUser = { send: jest.fn() };
+    spyDiscordClientUsers.mockImplementation(() => mockDiscordUser);
+
+    await sendTo('BalBalBlub', new TokenRequestMessage(), { key: 123123 });
```

Comment:
```suggestion
    await sendTo('BlaBlaBlub', new TokenRequestMessage(), { key: 123123 });
```
Um dich zu ärgern :P